docs/appendices/glossary.rst

```diff
@@ -79,20 +79,51 @@ M
 
 .. _gloss-n:
 
+N
+-
+
+.. _gloss-nonscalar:
+
+**Nonscalar**
+    A :ref:`data type <data-types>` that can have multiple component values
+    (e.g., :ref:`arrays <data-type-array>` and :ref:`objects
+    <object_data_type>`).
+
+    Contrary to a :ref:`scalar <gloss-scalar>`.
+
+    .. SEEALSO::
+
+        :ref:`sql_ddl_datatypes_geographic`
+
+        :ref:`sql_ddl_datatypes_compound`
```

Comment:
okay! changed it to "container types". note that the term "compound types" was already in use and I just linked to it. but we can do the change as a part of this commit

I have kept the term "nonscalar" in the glossary because we distinguish different types of nonscalar data types: containers (array and object) and geospatial data types

Reply:
Thank you.